### PR TITLE
workaround for kubeadmin

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

**Related Issue:**  stolostron/backlog#<ISSUE_NUMBER>

### Description of changes
Skipping RBAC temporarily for kubeadmin

### CODE REVIEW

File: pkg/resolver/search.go [hunk 1]
```diff
@@ -112,10 +114,27 @@ func (s *SearchResult) Related() []SearchRelatedResult {
...
+			break
+		}
+
+	}
+	if turnoffRbac || userDetails.Username == "kube:admin" {
+		klog.Warning("TEMPORARY WORKAROUND for Kubeadmin: Turning off RBAC")
+		return true
+	}
+	return false
+}
```
Comment (jlpadilla) [inline pkg/resolver/search.go:136]:
```suggestion
func Iskubeadmin(ctx context.Context) bool {
	_, userDetails := rbac.CacheInst.GetUserUID(ctx)
	if userDetails.Username == "kube:admin" {
		klog.Warning("TEMPORARY WORKAROUND for Kubeadmin: Turning off RBAC")
		return true
	}

	for _, group := range userDetails.Groups {
		if group == "system:cluster-admins" {
			return true
		}
	}	
	return false
}
```

Comment (SherinV) [reply pkg/resolver/search.go]: Updated

File: pkg/resolver/search.go [hunk 2]
```diff
@@ -112,10 +114,24 @@ func (s *SearchResult) Related() []SearchRelatedResult {
...
+func Iskubeadmin(ctx context.Context) bool {
+	_, userDetails := rbac.CacheInst.GetUserUID(ctx)
+	if userDetails.Username == "kube:admin" {
+		klog.Warning("TEMPORARY WORKAROUND for Kubeadmin: Turning off RBAC")
+		return true
+	}
+	for _, group := range userDetails.Groups {
+		if group == "system:cluster-admins" {
+			return true
+		}
```
Comment (jlpadilla) [inline pkg/resolver/search.go:130]:
```suggestion
	for _, group := range userDetails.Groups {
		if group == "system:cluster-admins" {
			klog.Warning("TEMPORARY WORKAROUND for Kubeadmin: Turning off RBAC")
			return true
		}
```

**Review by jlpadilla — APPROVED** [review]
/lgtm

Merging this workaround to stabilize the canary builds after switching to V2. We will remove and implement correctly ASAP.

**Pull request merged**